app/src/main/java/io/dolby/rtsviewer/ui/streaming/StreamingScreen.kt

```diff
@@ -80,6 +76,21 @@ fun StreamingScreen(viewModel: StreamingViewModel = hiltViewModel(), onBack: ()
                         viewModel.pauseVideo()
                     }
                 }
+                DisposableEffect(uiState.videoTrack, uiState.pendingSelectedStreamQualityType) {
```

Comment:
Add an effect for changed video tracks or selected stream quality that only does resuming